### PR TITLE
Updated learner analytics breadcrumb message.

### PR DESCRIPTION
**Pull request by dsjen**

### CODE REVIEW

File: analytics_dashboard/courses/views/__init__.py [hunk 1]
```diff
@@ -530,23 +530,29 @@ def get_table_items(self):
...
-                        # TODO: what would the breadcrumbs be?
-                        'breadcrumbs': [_('Learners')]
-                    },
-                    {
-                        'title': _("Who hasn't watched videos recently?"),
-                        'view': 'courses:learners:learners',  # TODO: map this to the actual action in AN-6205
-                        # TODO: what would the breadcrumbs be?
-                        'breadcrumbs': [_('Learners')]
-                    }
+                    # TODO: this is commented out until we complete the deep linking work, AN-6671
```
Comment (dsjen) [inline analytics_dashboard/courses/views/__init__.py:537]: If you prefer, I can remove the commented out lines.  I always feel a little weird commenting out code, but I figure that we'll eventually include this again, although I guess we can always look back in the history to retrieve it.  Let me know what you think!

**Comment by dsjen** [comment]
@dan-f please review

File: analytics_dashboard/courses/views/__init__.py (hunk 1 above)
Comment (dan-f) [inline analytics_dashboard/courses/views/__init__.py:537]: Yeah, fair point. I don't mind if you keep it or remove it :)

Comment (dsjen) [inline analytics_dashboard/courses/views/__init__.py:537]: Is that a thumbs up? :pray:

Comment (dan-f) [inline analytics_dashboard/courses/views/__init__.py:537]: Totally. :+1:

**Comment by dan-f** [comment]
:cake:

**Pull request merged**